pkg/ctl/utils/enable_logging.go

```diff
@@ -103,29 +78,32 @@ func doEnableLogging(rc *cmdutils.ResourceCmd) error {
 		return err
 	}
 
-	shouldEnable := sets.NewString()
-
+	willBeEnabled := sets.NewString()
 	if cfg.HasClusterCloudWatchLogging() {
-		shouldEnable.Insert(cfg.CloudWatch.ClusterLogging.EnableTypes...)
+		willBeEnabled.Insert(cfg.CloudWatch.ClusterLogging.EnableTypes...)
+	} else {
+		baselineEnabled := sets.NewString(currentlyEnabled.List()...).Union(willBeEnabled)
```

Comment:
Minor nit: the `Union` is a no-op since `willBeEnabled` will always be an empty set here.

Reply:
yeah, I think i was tweaking this several times and now it didn't make sense anymore :)